presentation-api/controlling-ua/defaultRequest_success-manual.html

```diff
@@ -9,34 +9,52 @@
 <script src="common.js"></script>
 
 <p>Click the button or the menu item for presentation on your browser (for example, "Cast"), to start the manual test.</p>
```

Comment:
I suggest to complete with "and select a presentation display when prompted to do so. The test passes if a "PASS" result appears" to clarify what the user needs to do.